### DISCUSSION TITLE
Reintroducing Volumetric Nebula

### DISCUSSION THREAD

**Discussion by Jorpis-Jorah** [tsunamayo/Starship-EVO#4135] [Suggestions]
Y'all remember Nebulae being in the game? They were cool. Sadly removed after the name changed. 
I'm hoping that in the future they are reintroduced in the game. They were lots of fun to travel to and explore. I think that nebulae gave player more reasons to explore the galaxy and find other points of interest. I hope they are added again, but not too soon. I don't want you to stop working on what you are currently focused on, this is just something that you could keep in mind when the time comes.
![image](https://user-images.githubusercontent.com/85460804/130994781-45846e24-592e-4aea-8e32-8cdcffd14bbb.png)

**Comment by Briaireous** [comment]
Hold up, you were able to fly to those? I thought they were just part of the skybox

**Reply by Jorpis-Jorah** [reply]
yep

**Reply by Jorpis-Jorah** [reply]
> Hold up, you were able to fly to those? I thought they were just part of the skybox

Well, actually they were technically part of the skybox, but yes, they were real.
